.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Task
+about: When a user story has to be broken into smaller engineering chunks, use a task!
+title: 'Task: Part of #Issue'
+labels: ''
+assignees: Sgtpluck
+
+---
+
+This task is part of #71
+
+Full User Story:
+As a Program Specialist
```

Comment:
[suggestion] It seems fine to use a realistic example, but if we wanted to match the other templates you could just copy the user-story example entirely here (`Given that I am a [type of user]`)

Reply:
yeah that's a good idea